### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by somexpert**
This will add/register the admin pages for django-citeit and consequently bump the version to 1.1.0

### CODE REVIEW

**Comment by somexpert** [comment]
@ldko, @damonkelley, this is ready for review.

File: citeIt/site_views.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+from citeIt.models import Location as citeIt_Location
+from citeIt.models import Institution as citeIt_Institution
+#citeIt Admins
+from citeIt.admin import DegreeLevelAdmin, CitationAdmin
+from citeIt.admin import InstitutionAdmin as citeIt_InstitutionAdmin
+#Register all the admin objects that live in the main admin
+main_admin = admin.AdminSite(name="main")
+#Auth
+main_admin.register(User, UserAdmin)
+main_admin.register(Group, GroupAdmin)
```
Comment (damonkelley) [inline citeIt/site_views.py]: We don't need to register the users and groups

File: citeIt/site_views.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+from django.contrib.auth.admin import UserAdmin, GroupAdmin
+#citeIt Models
+from citeIt.models import DegreeLevel, Subject, Citation
+from citeIt.models import Location as citeIt_Location
+from citeIt.models import Institution as citeIt_Institution
+#citeIt Admins
+from citeIt.admin import DegreeLevelAdmin, CitationAdmin
+from citeIt.admin import InstitutionAdmin as citeIt_InstitutionAdmin
+#Register all the admin objects that live in the main admin
+main_admin = admin.AdminSite(name="main")
```
Comment (damonkelley) [inline citeIt/site_views.py]: Does this need to be here?

File: citeIt/site_views.py (hunk 1 above)
Comment (somexpert) [inline citeIt/site_views.py]: Ah, I was wondering about that. I'll remove it.

**Comment by damonkelley** [comment]
I don't think we need to add `site_views.py` just to register the Admin models. We can do that from `admin.py`. Also, check this out. https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#the-register-decorator

**Comment by somexpert** [comment]
Yep, you are completely correct. In fact, they were already registered. Well, most of this was pointless, I should have looked more into it before assuming that site_views.py was needed. Thanks though, now I know what to look for.

**Pull request closed without merging**